Dockerfile

```diff
@@ -35,7 +35,7 @@ ENV PKGS_LIST=main-packages-list.txt
 ARG EXTRA_PKGS_LIST
 
 COPY ${PKGS_LIST} ${EXTRA_PKGS_LIST} /tmp/
```

Comment:
A previous version of this PR included "${PATCH_LIST}" in this line, but its now gone, the weird thing is that it still works, this brought me down a bit of a rabbit hole. As far as I can see leaving "EXTRA_PKGS_LIST" blank results in the copy command treating it as a "*", so the entire contents of the local directory get copied into /tmp (including my PATCH_LIST file). So it works but isn't what we would want as the /tmp directory is getting cluttered in the image.

Reply:
@derekhiggins we saw this behavior in other situations and this is actually a bug in podman, although I can't find the exact reference now
the ${PATCH_LIST} is actually needed here, thanks for catching this